### ISSUE TITLE
Glosses in analyzed wordlists contain translations

### ISSUE THREAD

**Issue by vidraj** [timarkh/uniparser-grammar-eastern-armenian#1]
It seems that the `gloss` attribute in [wordlists/eanc_words_analyzed.txt](https://github.com/timarkh/uniparser-grammar-eastern-armenian/blob/8852eb3659108e693eb7921a0d341d02d00ffeba/wordlists/eanc_wordlist_analyzed.txt) erroneously contain parts of the `trans_en` attribute instead of the STEM mark. Only when the translation is missing does the STEM appear.

See for example:
```
<w><ana lex="առաջ" gr="POST,sg,obl,def" parts="առաջ-ի-ն" gloss="before-OBL-DEF" trans_en="before, front side, ahead"></ana><ana lex="առաջի" gr="A,sg,nom,def" parts="առաջի-ն" gloss="STEM-DEF" trans_en=""></ana><ana lex="առաջին" gr="NUM,A,sg,nom,nonposs" parts="առաջին" gloss="first" trans_en="first"></ana>առաջին</w>
```

**Comment by timarkh** [comment]
This is actually a feature rather than a bug. The value of the ``gloss`` attribute is primarily targeted at (non-computational) linguists, abd that's what they expect to see there (see [Leipzig glossing rules](https://www.eva.mpg.de/lingua/resources/glossing-rules.php)). Note that it does not coincide with the translation: it's much shorter and does not contain whitespaces or hyphens. This means that ``gloss`` values have to be filled separately from ``trans_en`` in the dictionary. That takes time, which is why we only made glosses for 6,000 or so most frequent lemmata. The rest get a ``STEM`` placeholder instead (even if ``trans_en`` is not empty, which is true of many more items than those 6,000).

If you need the same gloss for each stem for some computational purposes, I suggest that you replace with ``STEM`` all character sequences that do not contain a hyphen and are not part of any ``gloss`` value inside ``paradigms.txt``. Or I could add a parser option that puts ``STEM`` across the board. If you need something like that, please let me know.

**Comment by vidraj** [comment]
Thank you for the answer! I've seen this feature only in the Eastern Armenian data, not in Udmurt or Komi-Zyrian, so I thought it was an error, especially since the gloss sometimes contains quotes, breaking the XML structure (`democrat_member.of."Ramkavar".party.(Armenian.Liberal.Democratic.Party)`).

**Comment by timarkh** [comment]
You are right, Eastern Armenian is the only analyzer for which there was someone ready to invest their time into turning translations into glosses.

Oh wait, it seems my answer was not entirely correct. We did manually spell out glosses for some frequent words, but we also produced ones automatically from the translations for the rest. Only when ``trans_en`` value was empty did we leave the STEM placeholder (because a gloss should not be empty). So you were right saying that ``STEM`` only appears when the translation is missing. It was done a long time ago, so I don't remember all the details clearly, sorry about that.

Regarding quotes breaking XML: *that* is actually a bug, which I was unaware of. Probably I missed some character escaping somewhere in the pipeline. I'll try to fix that in the next days, thanks a lot for reporting!
